hadoop-ozone/ozone-manager/src/test/java/org/apache/hadoop/ozone/om/request/key/TestOMKeyCreateRequest.java

```diff
@@ -414,96 +411,14 @@ private OMRequest createKeyRequest(boolean isMultipartKey, int partNumber,
         .setCreateKeyRequest(createKeyRequest).build();
   }
 
-  @Test
-  public void testKeyCreateWithFileSystemPathsEnabled() throws Exception {
```

Comment:
This test is no longer valid for LEGACY / OBJECT_STORE case, so I have moved it to the FSO variant of this test suite.